LiquidCooling/OCPCoolantDistributionUnit.v1_0_0.json

```diff
@@ -0,0 +1,208 @@
+{
+    "SchemaDefinition": "RedfishInteroperabilityProfile.v1_7_0",
+    "ProfileName": "OCP Coolant Distribution Unit",
+    "ProfileVersion": "1.0.0",
+    "Purpose": "Specifies the OCP management requirements for the Redfish interface on OCP coolant distribution units.",
+    "OwningEntity": "Open Compute Project",
+    "ContactInfo": "michael.raineri@dell.com",
+    "RequiredProfiles": {
+        "OCPLiquidCoolingBaseline": {
+            "MinVersion": "1.0.0"
+        }
+    },
+    "Resources": {
+        "ThermalEquipment": {
+            "URIs": [
+                "/redfish/v1/ThermalEquipment"
+            ],
+            "PropertyRequirements": {
+                "CDUs": {}
+            }
+        },
+        "CoolingUnit": {
+            "URIs": [
+                "/redfish/v1/ThermalEquipment/CDUs/{CoolingUnitId}"
+            ],
+            "PropertyRequirements": {
+                "CoolantConnectorRedundancy": {
+                    "ReadRequirement": "IfImplemented"
+                },
+                "EquipmentType": {
+                    "Comparison": "Equal",
+                    "Values": [
+                        "CDU"
+                    ]
+                },
+                "FilterRedundancy": {
+                    "ReadRequirement": "IfImplemented"
+                },
+                "Filters": {},
+                "PrimaryCoolantConnectors": {},
+                "PumpRedundancy": {
+                    "ReadRequirement": "IfImplemented"
+                },
+                "Pumps": {},
+                "Reservoirs": {
+                    "ReadRequirement": "IfImplemented"
+                },
+                "SecondaryCoolantConnectors": {},
+                "Status": {
+                    "PropertyRequirements": {
+                        "Health": {},
+                        "State": {}
+                    }
+                }
+            }
+        },
+        "Filter": {

```

Comment:
Will need to make this "IfImplemented"; speaking with some CDU folks, there are some designs that simply don't have filters.

Reply:
Made change for this; leaving open as a remind for others to review to ensure agreement.

Reply:
Agreed